flowmachine/flowmachine/features/utilities/feature_collection.py

```diff
@@ -3,13 +3,13 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 """
-Class definition for FeatureCollection, this is a group of
+Class definition for feature_collection, this is a group of
 joined features.
 """
-from ...core import Query
+from flowmachine.core.join import Join
 
 
-class FeatureCollection(Query):
+def feature_collection(metrics, dropna=True) -> Join:
```

Comment:
Mainly out of interest (certainly not against this): what made it necessary / desirable to turn `feature_collection` into a function rather than leaving it as a class?

Reply:
It really just wraps a bunch of calls to Join, and doesn’t really have any relevant state of its own that means it _needs_ to be a class, is my thinking here.

Reply:
Makes sense. I just saw that because of the conversion the `Returns` section is missing in the docstring. Can we add this please?